data/json/recipes/basecamps/recipe_modular_livestock_log.json

```
@@ -0,0 +1,77 @@
+    "result": "faction_base_modular_livestock_coop_log_northeast",
+    "description": "We could use a coop to contain our chickens and other birds, so build a log shack with a roof on the northeast side of the livestock field.",
+    "category": "CC_BUILDING",
+    "subcategory": "CSC_BUILDING_BASES",
+    "autolearn": false,
+    "never_learn": true,
+    "construction_blueprint": "fbml_coop_log_northeast",
+    "blueprint_name": "northeast coop",
+    "blueprint_requires": [ { "id": "fbml_0" } ],
+    "blueprint_provides": [ { "id": "fbml_northeast" }, { "id": "chicken_coop" } ],
+    "blueprint_excludes": [ { "id": "fbml_northeast" } ],
+    "blueprint_autocalc": true
+  },
+  {
+    "type": "recipe",
+    "result": "faction_base_modular_livestock_shack_log_southeast",
+    "description": "We could use a storage room for feed, animal gear and tools, so build a log shack with a roof on the southeast side of the livestock field.",
+    "category": "CC_BUILDING",
+    "subcategory": "CSC_BUILDING_BASES",
+    "autolearn": false,
+    "never_learn": true,
+    "construction_blueprint": "fbml_shack_log_southeast",
+    "blueprint_name": "southeast storage room",
+    "blueprint_requires": [ { "id": "fbml_0" } ],
+    "blueprint_provides": [ { "id": "fbml_southeast" } ],
+    "blueprint_excludes": [ { "id": "fbml_southeast" } ],
+    "blueprint_autocalc": true
+  },
+  {
+    "type": "recipe",
+    "result": "faction_base_modular_livestock_stable0_log_southwest",
+    "description": "We could use a stable for larger livestock, so build a log shack with a roof on the southwest side of the livestock field.",
+    "category": "CC_BUILDING",
+    "subcategory": "CSC_BUILDING_BASES",
+    "autolearn": false,
+    "never_learn": true,
+    "construction_blueprint": "fbml_stable0_log_southwest",
+    "blueprint_name": "southwest stable",
+    "blueprint_requires": [ { "id": "faction_base_livestock_0" } ],
+    "blueprint_provides": [ { "id": "fbml_southwest" }, { "id": "stables" } ],
+    "blueprint_excludes": [ { "id": "fbml_southwest" } ],
+    "blueprint_autocalc": true
+  },
+  {
+    "type": "recipe",
+    "result": "faction_base_modular_livestock_stable1_log_west",
+    "description": "We could use an expansion for the stable, so build 2 more stables on the west side of the livestock field.",
+    "category": "CC_BUILDING",
+    "subcategory": "CSC_BUILDING_BASES",
+    "autolearn": false,
+    "never_learn": true,
+    "construction_blueprint": "fbml_stable1_log_west",
+    "blueprint_name": "west stable",
+    "blueprint_requires": [ { "id": "fbml_southwest" } ],
+    "blueprint_provides": [ { "id": "fbml_west" } ],
+    "blueprint_excludes": [ { "id": "fbml_west" } ],
+    "blueprint_autocalc": true
+  },
+  {
+    "type": "recipe",
+    "result": "faction_base_modular_livestock_stable2_log_northwest",
+    "description": "We could use an expansion for the stable, so build 2 more stables on the northwest side of the livestock field.",
```

```suggestion
    "description": "We could use an expansion for the stable, so build 2 more stalls on the northwest side of the livestock field.",
```

You're expanding the stable with more stalls, not with more stables.